changelog.d/297.fixed.md

```diff
@@ -0,0 +1 @@
+Fix PM matching wrongly using ifdescr instead of the correct ifalias
```

Comment:
Please update this to be more concrete on when `ifalias` is used for matching

Reply:
I can grok both versions, so resolving this. As release manager, I reserve the right to rewrite any and all changelog entries as I see fit before committing the compiled changelog :)